test/e2e/suite/smoke-tests/addons_test.go

```diff
@@ -0,0 +1,110 @@
+//go:build e2e
+// +build e2e
+
+package addons
```

Comment:
i dont fully get the `smoke_tests` folder here? why isnt this just the addons_test?